webapps/ui/cockpit/plugins/base/app/views/processInstance/jobRetryBulkDialog.js

```diff
@@ -36,8 +38,21 @@ module.exports = [
     $modalInstance,
     processData,
     processInstance,
-    $translate
+    $translate,
+    fixDate
   ) {
+    $scope.radio = {value: 'preserveDueDate'};
+    $scope.dueDate = moment().format('YYYY-MM-DDTHH:mm:00');
+
+    $scope.checkDateFormat = date =>
+      /(\d\d\d\d)-(\d\d)-(\d\d)T(\d\d):(\d\d):(\d\d)(?:.(\d\d\d)| )?$/.test(
```

Comment:
❓ I see this format in multiple places, not just here but in other JS files too. Maybe we should add some reusable utility component?

Reply:
Good idea. Since I want to make progress on this PR (refactoring this means some additional testing effort), I created a follow-up ticket: #3210.